support/db/user.toml

```diff
@@ -1,3 +1,3 @@
-listen_addresses = "127.0.0.1"
-search_path = "public"
-hba_file = "/hab/svc/postgresql/pg_hba.conf"
```

Comment:
And I realize now I failed to mention that this PR was to get our tests working again... apologies for the confusion @eeyun !